pkg/compose/port.go

```diff
@@ -26,18 +26,40 @@ import (
 	moby "github.com/docker/docker/api/types"
 )
 
-func (s *composeService) Port(ctx context.Context, projectName string, service string, port uint16, options api.PortOptions) (string, int, error) {
+func (s *composeService) Port(ctx context.Context, projectName string, service string, port uint16, options api.PortOptions) (api.PortPublishers, error) {
```

Comment:
as signature changes, and breaks compatibility, makes sense to _also_ rename `Ports`(plural)

Reply:
@ndeloof Sorry to bother but do you have any thoughts on my comments? Should we still perform the renaming?